devops/scripts/benchmarks/benches/umf.py

```diff
@@ -208,20 +186,56 @@ def parse_output(self, output):
             raise ValueError("Benchmark output does not contain data.")
 
         results = []
+
         for row in reader:
             try:
                 full_name = row[self.col_name]
                 pool, config = self.get_pool_and_config(full_name)
-                mean = self.get_mean(row)
-                updated_pool = self.get_preloaded_name(pool)
-                updated_config = self.get_preloaded_name(config)
+                statistics = None
+                is_row_matched_to_statistics_type = False
+
+                if not self.is_memory_overhead_checked:
+                    statistics = self.get_mean(row)
+
+                    is_row_matched_to_statistics_type = True
+
+                    # At this moment, preloaded benchmarks
+                    # do not support memory statitics
+                    if self.is_preloaded:
+                        pool = self.get_preloaded_pool_name(pool)
+
+                elif self.is_memory_statistics_included(row):
+                    statistics = self.get_memory_overhead(row)
+                    config = "FRAGMENTATION_" + config
+
+                    is_row_matched_to_statistics_type = True
+
+                if is_row_matched_to_statistics_type:
+                    results.append((config, pool, statistics))
```

Comment:
You can simplify this by just appending results twice - once with mean and once with fragmentation. And then you will be able to remove the `is_memory_overhead_checked` variable and `GBenchMemoryOverhead` class.